Trakt/Helpers/LibraryManagerEventsHelper.cs

```diff
@@ -137,6 +137,21 @@ private async Task OnQueueTimerCallbackInternal()
                     _logger.LogInformation("No movie additions to process");
                 }
 
+                var queuedMovieUpdates = queue.Where(ev =>
+                    new Guid(ev.TraktUser.LinkedMbUserId).Equals(new Guid(traktUser.LinkedMbUserId)) &&
```

Comment:
`new Guid(traktUser.LinkedMbUserId)` is being called a lot. It's just a struct, but it should really be saved in a variable

Reply:
And also, the queue is being iterated quite a few times. Would it not be better to iterate once and add events to separate lists?